internal/operators/manager.go

```diff
@@ -206,7 +207,7 @@ func (mgr *Manager) GenerateManifests(ctx context.Context, cluster *common.Clust
 // so user can provide here even CRs provisioned by the OLM.
 func (mgr *Manager) createControllerManifest(ctx context.Context, cluster *common.Cluster, content string) error {
 	objectFileName := path.Join(string(*cluster.ID), controllerManifestFile)
-	if err := mgr.objectHandler.Upload(ctx, []byte(content), objectFileName); err != nil {
+	if err := mgr.objectHandler.Upload(ctx, []byte(content), objectFileName, map[string]string{constants.ManifestSourceAttribute: constants.ManifestSourceSystemGenerated}); err != nil {
```

Comment:
Really just behaviour that has not been changed as part of any PR. 

It would make more sense to migrate this to the manifest API. 

I do notice that the object path is outside of the "traditional" manifest paths. 

In this case the "manifest" is stored in `{cluster id}/{file name}` as opposed to `{cluster id}/manifests/openshift/{file name}`

Everything about this file seems to suggest it is a manifest but there are unanswered questions.

I will look into whether we can put this in manifests/openshift alongside all the other manifests.

The good news is that if we are going down the path of dedicated metadata functions, any changes here would not be required for now.

Reply:
I decided in the end to pass `nil` for manifest metadata in this case.

This is because (in spite of naming convention) this resides outside of the manifest upload feature.

I am asking @rccrdpccl about this in a separate conversation regarding the motivation for implementing in this way. Depending on the outcome of that conversation, a PR may be created.